src/symbolize/gimli/mmap_unix.rs

```diff
@@ -29,6 +29,22 @@ impl Mmap {
         }
         Some(Mmap { ptr, len })
     }
+
+    #[cfg(target_os = "android")]
+    pub unsafe fn map_with_offset(file: &File, len: usize, offset: usize) -> Option<Mmap> {
```

Comment:
Change `Mmap::map` instead. Alter callers to pass the 0 offset, and make the offset a 64-bit integer, not a usize, like [Seek](https://doc.rust-lang.org/std/io/trait.Seek.html) uses.

Reply:
I can definitely do that; my original reason for creating a separate method was to avoid modifying the fake + Windows mmap implementations to match.

Reply:
It's alright if you leave the extra parameter unused and leave a FIXME behind for those.

Reply:
I ended up implementing them anyway, didn't seem too difficult (besides Windows's more-than-page-size alignment requirements which tripped up my testing lol).